### DISCUSSION TITLE
OPError: expected 200 OK, got: 302 Found

### DISCUSSION THREAD

**Discussion by criffin123** [panva/openid-client#633] [Q&A]
How can I resolve such scenario when authorize endpoint returns 302 with location redirect to login page? 
Is there a way to enable logging to console for the openid-client calls and responses?

I'm using the module via passport-curity and calling it as following: 

    const client = await discoverAndCreateClient({
        issuerUrl: process.env.ONELOGIN_AUTHORIZE_ENDPOINT,
        clientID: process.env.CLIENT_ID,
        clientSecret: process.env.CLIENT_SECRET,
        clientSecret: process.env.CLIENT_SECRET,
        redirectUris: [process.env.ONELOGIN_REDIRECT_URL_SUCCESS],
        responseTypes: ['code', 'token']
    });

**Comment by panva** [comment]
The issuerUrl is not supposed the authorize endpoint. Please refer to your issuer's documentation to learn what the correct Issuer Identifier is.
